core/src/sumeragi/main_loop.rs

```diff
@@ -662,13 +676,33 @@ fn process_message_independent(
                         #[cfg(debug_assertions)]
                         if is_genesis_peer && sumeragi.debug_force_soft_fork {
                             std::thread::sleep(sumeragi.pipeline_time() * 2);
-                        } else {
+                        } else if current_view_change_index >= 1 {
                             sumeragi.broadcast_packet(msg);
+                        } else {
+                            sumeragi.broadcast_packet_to(
+                                msg,
+                                current_topology
+                                    .sorted_peers
+                                    .iter()
+                                    .zip(0..current_topology.min_votes_for_commit())
+                                    .map(|(p, _)| p),
+                            );
```

Comment:
This code will try to send message to the peer itself. We should skip sender.

Reply:
I have added a check in the post_packet_to function.